plugins/field-colour/src/index.ts

```diff
@@ -5,3 +5,4 @@
  */
 
 export * from './field_colour';
+export * from './blocks';
```

Comment:
Same as for imports:

```suggestion
export * from './field_colour.js';
export * from './blocks.js';
```

… except, wait: there is no `blocks.ts` or `blocks.js` in this PR, as far as I can see.  Is there some magic going on here, where `export * from 'dirname'` creates namespaces for the exports from each file in `dirname`???

Reply:
I think there's magic where having an `index.ts` means you can access it through the directory name. The re-exports from each file in `blocks/` are done in `blocks/index.ts`, so there's no magic there.

Reply:
Oh, OK.  I had not paid close enough attention to the file structure.

I think we should avoid relying on such magic, because it is almost certainly provided by webpack.  AFIAK browsers (absent an import map) only ever import the exact file that appears in the import statement.  That means that, at such point as we start publishing the `.js` versions of the source files the current layout will break.

The straightforward fix is to change the line to `export * from './blocks/index.js';`, but I think a better approach would be to fold the contents of `blocks/index.ts` directly into `src/index.js`; AFAICT the only change needed would be to update the relative paths in the `import` and `export … from` statements.  This gives the top level `index.js` file something useful to do and eliminates at least one round-trip when loading the package unbundled.  (If you want to leave `installAllBlocks` in `blocks/index.js` then I would still recommend moving the `export … from` statements to the top-level `index.js` to improve loading parallelism.)

Reply:
Exports moved for `src/index.ts` and file deleted.